resources/assets/components/Post/index.js

```diff
@@ -59,6 +59,7 @@ class Post extends React.Component {
     const user = this.props.user ? this.props.user : null;
     const signup = this.props.signup;
     const campaign = this.props.campaign;
+    let quantity = post.quantity != null ? post.quantity : 0;
```

Comment:
Could this be `const`?

Reply:
yes I think so! I just wasn't sure since it could change?

Reply:
doesn't look like it ever gets changed so I think it'd be good as `const`!

Reply:
hrrrmmmm i think it could be `const` ! it doesn't change throughout the lifecycle of the component.

Reply:
makes sense! updated :)